rero_ils/dojson/cli.py

```diff
@@ -18,6 +18,8 @@
 """Click command-line utilities."""
 
 
+import json
```

Comment:
why?

Reply:
without this import, the generation of documents will not work